packages/app/src/components/pages/Home/useSort.js

```diff
@@ -55,11 +56,14 @@ const useSort = () => {
     return sortableNoms
   }, [NominationsData, sortConfig])
 
-  const requestSort = (key) => {
-    let direction = 'ascending'
-
-    if (sortConfig && sortConfig.key === key && sortConfig.direction === 'ascending') {
-      direction = 'descending'
+  const requestSort = (key, doDefault) => {
+    console.log(doDefault)
```

Comment:
Let's remove this

Reply:
Sure thing, that would be the console statement?

Reply:
Yep, just the console log

Reply:
Done.